schema/relationship-record.json

```
@@ -33,7 +33,7 @@
     },
     "interestedParty": {
       "title": "Interested party",
-      "description": "The interested party has some level of ownership or control over the entity referenced by `subject` in this relationship record. The value is a record Id for either an entity record or person record, or, where the interested party is unknown, details of why. When confirmation has been provided that no interested party exists, where ownership and control information does not need to be provided, or where details of ownership and control are unknown, a `reason` MUST be given. Where an unknown entity is the `subject` of further Relationship Records in the same structure, or where there is a natural person with ownership or control but their name or details are not known or cannot be disclosed for some reason, a reference recordId for a person or entity should be provided but identifying details in the record MAY be left blank.",
+      "description": "The recordId for the interested party in the relationship, or a reason why the interested party cannot be specified. The interested party MAY be an entity or a person. Where the entity or person is unknown or anonymous, the recordDetails of the interested party SHOULD reflect this. In place of a recordId, a reason SHOULD be given in cases where interested parties beyond this point of the beneficial ownership network: cannot be disclosed, have not been disclosed, are not required to be disclosed, or do not exist.",
```

```suggestion
      "description": "The recordId for the interested party in the relationship, or a reason why the interested party cannot be specified from the Unspecified Reason Codelist. The interested party MAY be an entity or a person.",
```
'Where the entity or person is unknown or anonymous, the recordDetails of the interested party SHOULD reflect this.' I think this contradicts the guidance in the BO page which implies there would be no record for the interested party 

'If an entity declares that it has no disclosable beneficial owners, its record MUST be the subject of a Relationship statement with recordDetails.interestedParty.reason ‘noBeneficialOwners’.'

---

There's a bit of subtlety that your succinct edit doesn't capture, but we need to retain for now. I've tried a further edit - see what you think.